### PR TITLE
Glossary

### PR DESCRIPTION
**Pull request by gottacatchenall**
an empty glossary at the end of manuscript md

### CODE REVIEW

**Comment by gottacatchenall** [comment]
it's possible we will want to assign specific definitions to specific people, not sure how we want to do that to avoid merge conflicts though

**Comment by TanyaS08** [comment]
There's a part of me that wants to just relegate flagging terms needed for the glossary to the comments/an issue and then we define and collate later (but that's probably creating unneeded work).  The idea of assigning a specific term to a person could work... Then its just down to internal comms to co-ordinate but I like the proposal of doing something similar to what we are doing with the author details in the metadata file

**Comment by gottacatchenall** [comment]
> There's a part of me that wants to just relegate flagging terms needed for the glossary to the comments/an issue and then we define and collate later (but that's probably creating unneeded work). 

I agree, I'll make a glossary issue and we can tag people and explain there

File: manuscript.md [hunk 1]
```diff
@@ -66,4 +66,22 @@
 
 ## What ecological knowledge would forecasting bring?
 
+
+## Glossary
```
Comment (tpoisot) [inline manuscript.md:70]: I would put this as a box -- markdown has a nice way to write definitions:

~~~
Metaweb
: hella many interactions
~~~

It doesn't work on github markdown, but this should get rendered in the PDF and html

**Review by tpoisot — CHANGES_REQUESTED** [review]

**Comment by tpoisot** [comment]
One thing we *could* do is `glossary.json` - this would be very easy to render in the template(s), and if this is self-contained to a file, it would make it easier to work on.

~~~ json
[
  {
    "term": "Metaweb",
    "definition": "A lot of interactions"
  },
  {
    "term": "Machine learning",
    "definition": "Essentially magic, but sad"
  },
  {
    "term": "Ecological network",
    "definition": "A community defined by people who wish they were better at maths"
  }
]
~~~

**Comment by gottacatchenall** [comment]
i like a json glossary idea a lot---assuming those definitions are included, of course

**Comment by TanyaS08** [comment]
Have to agree with @gottacatchenall 100% - a suggestion for interactions: how much two species vibe with each other

**Comment by tpoisot** [comment]
Predation :

ztHKpje

**Comment by gottacatchenall** [comment]
i believe this is called a "serval"

hejustvibin

**Comment by TanyaS08** [comment]
Hmm something looks off -but I can't quite put my finger on it

**Comment by gottacatchenall** [comment]
> Hmm something looks off -but I can't quite put my finger on it
bad vibes, probably

**Comment by TanyaS08** [comment]
Yeah does not pass the vibe check

**Comment by gottacatchenall** [comment]
please submit a vibe pull request and we'll take a look

**Comment by TanyaS08** [comment]
Flagging forbidden links 🙅‍♀️ here to potentially be included

**Pull request closed without merging**